src/vs/workbench/contrib/chat/browser/chatDragAndDrop.ts

```diff
@@ -184,3 +194,15 @@ function getFileAttachContext(resource: URI): IChatRequestVariableEntry | undefi
 		isDynamic: true
 	};
 }
+
+function getImageAttachContext(resource: URI, fileBuffer?: VSBuffer): IChatRequestVariableEntry | undefined {
+	const fileName = basename(resource);
+	return {
+		id: 'image',
```

Comment:
I think `id` needs to be a unique identifier for the specific image. This then allows us to make sure we only attach the same image once. Probably `id: resource.toString()` would work.